lib/stencil-bundle.js

```diff
@@ -154,19 +155,35 @@ class Bundle {
     assembleTemplatesTask(callback) {
         console.log('Template Parsing Started...');
 
-        recursiveReadDir(this.templatesPath, ['!*.html'], (readdirError, files) => {
-            if (readdirError) {
-                return callback(readdirError);
-            }
-
-            const partials = files.map((file) => {
+        // eslint-disable-next-line node/no-unsupported-features/es-builtins
+        Promise.allSettled([
+            recursiveReadDir(this.externalTemplatesPath, ['!*.html']),
```

Comment:
Will it work if the theme doesn't have external components library?

Reply:
code was updated. no issue here